### ISSUE TITLE
Add historical data chart

### ISSUE THREAD

**Issue by hnykda** [epidemics/covid#251]
This is a followup for #233 (so it's blocked by it) to actually visualize the data as a second chart (timeseries).

We'll want to add a _second_ graph (to the "index" page? or where?), which shows the timeline in a particular country up to that point.

**Comment by lagerros** [comment]
Two suggestions: 
* A single, very wide graph, which includes both the extrapolation but starts from confirmed cases (this will break on mobile probably, and we're making a choice about being primarily desktop oriented)
* Multiple columns (I still think this will probably be necessary as we keep adding on more information)

Worst case is probably to have a new graph underneath the current, which clashes with displaying explanations of the model

**Comment by mathijshenquet** [comment]
I think adding it in the same graph would be uninformative because of how tiny the current cases would look (unless you plot it in log scale, which is probably a bad default). 

Another issue is that there is severe under reporting of cases so the lines of the models would not match the datapoints. You could transform the datapoint by the estimated underreporting but that kind of goes against the spirit of combining predictions with observed datapoints.

I think its best to put it in a separate graph where we display: Reported active cases, estimated active cases inferred from the data, the model line. This could replace the 'Active Infections' section. Possibly add an indication to the prediction graph to show how they overlap.

**Comment by wolverdude** [comment]
So to confirm: this will be a separate chart or part of the same one?

I personally feel it would be more compelling to include lines for confirmed & estimated historical cases on the forecasted chart so that people can see the trend lines for themselves.

But for viewing interventions, it's definitely best to have a separate chart that is _logarithmically scaled_. It would be great to do something like timeline.js, but I doubt it can be integrated with Plotly, but I could implement something functional though not quite as compelling using Plotly.

**Comment by hnykda** [comment]
I think that up until now we didn't want to include the historical to forecasted charts because the information is likely not that useful together (as confirmed are much lower than true active).

I think that #193 and this one is workable and the log scale makes sense here. 

Regarding plotly vs. timeline.js. I believe that plotly with these two lines (deaths vs. confirmed) and then either:
1. scatter/circles with hover on what it is (like really just something like https://plotly.com/javascript/time-series/ or https://plotly.com/javascript/shapes/#circle with hover tool displaying the details/link about the intervention)
2. graph annotations: https://plotly.com/javascript/text-and-annotations/#multiple-annotations
3. "true" timeline: https://stackoverflow.com/a/50646793/2319172

would be just the right thing and easy to do (my top candidate is simply 1, maybe combined with 2)

**Comment by lagerros** [comment]
Hmm... I don't like 1., because it doesn't show you _how long_ the interventions lasted for. 

I think @jk3141 's sketch is the best style guidelines, but do understand if we'll have to make trade-offs due to what's feasible to build. 

![image](https://user-images.githubusercontent.com/21347592/77833018-71668a80-70f7-11ea-8140-57cbf9324561.png)

**Comment by hnykda** [comment]
That design is nice in theory but it that:
1. you have "the right amount" of interventions to display (as it doesn't scale very well with many items IMO)
2. you are able to tell the end of the intervention (how do you so for e.g.: "_The Government proposes that airlines be able to receive credit guarantees in 2020 amounting to a maximum of SEK 5 billion..._" or "_The People’s Bank of China provided an additional 500 billion yuan (roughly US$71 billion) of liquidity into the country’s financial system on Tuesday, following 1.2 trillion yuan (over US$170 billion) the previous day_"? )
3. my prior is that it's hard to do in any charting system - have you ever seen it somewhere?

**Comment by mathijshenquet** [comment]
> UI for interventions original plan. There are few basic types, upon click, more of the hierarchy is displayed
<sub>[View in Slack](https://epidemicforecasting.slack.com/archives/CV1F9NB5K/p1585433693062100)</sub>

**Comment by mathijshenquet** [comment]
> Hi, I have made a sketch of the dashboard like <@UV6G09Z39> was suggesting in <https://github.com/epidemics/covid/issues/193>
• I used the Oxford dataset as an example, but the code should work with our dataset too after some minimal tweaking. The severity of the measure implemented is a "traffic light" (green=no measures, yellow=some measures, red=lockdown-style measures). You can find it here : <https://github.com/lorenzozanisi/covid-viz/blob/master/covid-viz.ipynb> . If this can be of any help, I'm happy to keep working on it (the layout certainly needs improving). I'm using Python Bokeh
<sub>[View in Slack](https://epidemicforecasting.slack.com/archives/CV3KMT19V/p1585421226011800)</sub>

**Comment by hnykda** [comment]
Updated in the original description

**Comment by hazarane** [comment]
Replaced by #477
